crates/devolutions-log/README.md

```diff
@@ -0,0 +1,2 @@
+# devolutions-log
+Common code for logging in devolutions apps
```

Comment:
nitpick: Sorry for repeating myself, but to strictly it’s only for the services provided in this repository, and we do have other services that this crate is not supposed to be used for. A different wording could be used. But not really a big deal.